data-prepper-plugins/otel-trace-source/src/main/java/com/amazon/dataprepper/plugins/source/oteltrace/OTelTraceSourceConfig.java

```diff
@@ -3,13 +3,13 @@
 import com.amazon.dataprepper.model.configuration.PluginSetting;
 
 public class OTelTraceSourceConfig {
-    private static final String REQUEST_TIMEOUT = "request_timeout";
-    private static final String PORT = "port";
-    private static final String SSL = "ssl";
-    private static final String HEALTH_CHECK_SERVICE = "health_check_service";
-    private static final String PROTO_REFLECTION_SERVICE = "proto_reflection_service";
-    private static final String SSL_KEY_CERT_FILE = "sslKeyCertChainFile";
-    private static final String SSL_KEY_FILE = "sslKeyFile";
+    public static final String REQUEST_TIMEOUT = "request_timeout";
+    public static final String PORT = "port";
+    public static final String SSL = "ssl";
+    public static final String HEALTH_CHECK_SERVICE = "health_check_service";
+    public static final String PROTO_REFLECTION_SERVICE = "proto_reflection_service";
+    public static final String SSL_KEY_CERT_FILE = "sslKeyCertChainFile";
+    public static final String SSL_KEY_FILE = "sslKeyFile";
```

Comment:
These don't need `public` accessibility, can be left as "package private" by providing no keyword.